Exercises-1-Basic/README.md

```diff
@@ -1 +1,18 @@
-# Exercises 1 - Basic
\ No newline at end of file
+# Exercises 1 - Basic
```

Comment:
You should commit the actual java classes with the solution.

Reply:
commit the actual java classes with the solution??? I don't understand what are you talking about, can you please give me an example?

Reply:
![image](https://user-images.githubusercontent.com/46200614/152350226-2a21b616-a08b-450e-bf55-17730a6f7a9b.png)

You need to commit the java file itself where you are writing java code in eclipse.

I should be able to pull your branch and run your solution locally.

Reply:
check out this example PR that I created:
https://github.com/yevgenylvovski/Java-Practice/pull/10

On the most right tab you can see the changed files.
Ignore the `.gitignore` file and check only the java file.
This is how you need to commit your solutions, as java files, so I can right away run it in Eclipse.